### DISCUSSION TITLE
cockpit webservers logs

### DISCUSSION THREAD

**Discussion by skrowec** [cockpit-project/cockpit#18449] [Q&A] [answered]
Hello i have this project from school project , i need webservers logs , i found addon Logger but im having difficulty with downloading dis addon , is there any other way to get logs?
thank you for your time

**Comment by martinpitt** [comment, marked as answer]
The most popular webservers these days (Apache, nginx) don't log to the journal, but instead to their own files. E. g. /var/log/apache2/. Cockpit's Logs page only supports the journal, deliberately.

**Reply by ghost** [reply]
It's perhaps a preconception to think that only "systemd" may get access to standard IO devices configured in the /dev filesystem, unless systemd has become a /dev replacement and sometime frustrating type of "open source" software.
